spec/api-browser-window-spec.js

```diff
@@ -256,6 +256,14 @@ describe('BrowserWindow module', function () {
         w.loadURL(server.url)
       })
 
+      it('should support support base url for data urls', (done) => {
+        ipcMain.once('answer', function (event, test) {
+          assert.equal(test, 'test')
+          done()
+        })
+        w.loadURL('data:text/html,<script src="loaded-from-dataurl.js"></script>', {baseURLForDataURL: 'file://' + path.join(fixtures, 'api') + path.sep})
```

Comment:
Just curious, is the trailing `path.sep` needed here?

String could be rewritten as a template string:

```js
`file://${path.join(fixtures, 'api')}${path.sep}`
```